templates/Default/sitemap.xml.twig

```diff
@@ -3,7 +3,7 @@
 {% for dataset in datasets %}
     <url>
         <loc>{{ vanityurl('pelagos_app_ui_dataland_default', {'udi': dataset.udi}) }}</loc>
-        <lastmod>{{ dataset.modificationTimeStamp|date('Y-m-d') }}</lastmod>
+        <lastmod>{{ (dataset.modificationTimeStamp > sitemapMinDate) ? dataset.modificationTimeStamp|date('Y-m-d') : sitemapMinDate|date('Y-m-d') }}</lastmod>
```

Comment:
Consider using the twig function `date` to convert it a Datetime so it can be compared. Formats may vary.

Reply:
It is a datetime from the controller.

Reply:
> It is a datetime from the controller.

Yes, but sitemapMinDate is a `string`